core/serial_ftdi.c

```diff
@@ -38,8 +38,8 @@
 #endif
 
 #include "errorhelper.h"
-#define INFO(context, fmt, ...)	report_info(stderr, "INFO: " fmt, ##__VA_ARGS__)
-#define ERROR(context, fmt, ...)	report_info(stderr, "ERROR: " fmt, ##__VA_ARGS__)
+#define INFO(context, fmt, ...)	report_info("INFO: " fmt, ##__VA_ARGS__)
+#define ERROR(context, fmt, ...)	report_info("ERROR: " fmt, ##__VA_ARGS__)
```

Comment:
While the bugfix looks good, should we as well get rid of the `context` parameter where it still exists, seeing that this is not used any more? This should probably be done in a separate pull request though.

Reply:
I wonder if this parameter ever was used?

Reply:
I think that's a pattern from libdivecomputer, where the context is providing the sink for the log data.

Reply:
This code grew up in libdivecomputer but ended up in subsurface.

I'm not even sure if this code is in use in any of our builds any more, since we moved to the jni-wrapped java serial drivers on android.